problems/variables.js

```diff
@@ -1,65 +1,118 @@
+// // circumference = 2PI * radius // note to self
+// // console.log(Math.PI) //note to self
+// // console.log((2 * Math.PI) * radius) // note to self
+// let circumferenceResult = (Math.round(2 * 3.141592653589793) * radius)
+// // console.log("The circumference is" + " " + circumferenceResult + ".")
+// console.log(`The circumference is ${circumferenceResult}.`)
+// //  * Calculate the area based on the radius, and log "The area is `areaOfCircle`".
+// //  * Hint: https://www.w3schools.com/jsref/jsref_pi.asp
+// //area=PI r**2 // note to self
+// let areaOfCircle = (Math.round(3.141592653589793) * (radius**2))
+// // console.log("The area is" + " " + areaOfCircle + ".")
+// console.log(`The area is ${areaOfCircle}.`)
+
+
+// // ## Problem Four
+// // Temperature Converter:
+// //  * Store a celsius temperature into a variable.
+// let tempInCelsius = 1
+// //  * Convert it to fahrenheit and output "`tempInCelsius`°C is `tempInFahrenheit`°F".
+// // (celsius * (9/5)+32)=fahrenheit // note to self
+// let tempInFahrenheit = (tempInCelsius * (9/5) + 32)
+// //console.log(tempInCelsius + "°C" + " " + "is" + " " + tempInFahrenheit + "°F" + ".")
+// console.log(`${tempInCelsius}°C is ${tempInFahrenheit}°F.`)
+
+// //  * Now store a fahrenheit temperature into a variable.
+// let tempInFahrenheit = 34
+// //  * Convert it to celsius and output "`tempInFahrenheit`°F is `tempInCelsius`°C."
+// // celsius = ((5/9)*(fahrenheit-32)) // note to self
+// let tempInCelsius = (Math.round((5/9) * (tempInFahrenheit - 32)))
+// console.log(tempInFahrenheit + " " + "°F" + " " + "is" + " " + tempInCelsius + " " + "°C" )
 
 
 // ## Problem Five
-
 // Grades Calculator:
 //  * Store Alices's grade on a test to a variable
+// let alicesGrade = 80
+
 //  * Store Bob's grade on a test to a variable
+// let bobsGrade = 65
+
 //  * Store Cam's grade on a test to a variable
+// let camsGrade = 99
+
 //  * Find the average grade of all students
+// let allGrades = (alicesGrade + bobsGrade + camsGrade)
+// console.log(allGrades + " " + "is the total of all grades.")
+// let averageGrade = Math.round((alicesGrade + bobsGrade + camsGrade)/3) 
+// let averageGrade = Math.round(allGrades/3)
+// console.log(averageGrade + " " + "is the average grade.")
+
 //  * Store Dee's grade on a test to a variable
+// let deesGrade = 85
+
 //  * Find the average grade of all students
+// let classAverage = Math.round((deesGrade+allGrades)/4)
+// console.log(classAverage + " " + "is the class average.")
+
 //  * Print out if Dee's grade is higher than the class average
+// console.log(deesGrade + " " + "is Dee's grade which is higher than the class average.")
 
 // ## Problem Six
-
 // Find the last number:
 // * You are given a number a. Print the last digit of a.
 // * Example
 // * Input: 
 // * a = 123
 // * Output:
 // * 3
-
 // Hint:
 // Use the remainder % operator.
+//console.log(123%5)
```

Comment:
Have spaces around operators for better readability.

Reply:
Will do.